### ISSUE TITLE
Editing a node

### ISSUE THREAD

**Issue by JKatheder** [JKatheder/bisociation-net#20]
## Description
As a user I want to edit the node's title and description in order to formulate an idea.

## Acceptance
- [x] Every node should contain a description
- [x] When creating a project, a root node is created as well and it contains the description and project title as label
- [x] When hovering over a node: show the description as tooltip
- [x] "Edit Description" is available in the ContextMenu

Low Priority:
- [x] Root description should not be changed
- [x] Root label should not be changed
- [x] Char limit 1000 (as in data base)
- [x] Extra: Description for each other node
- [x] https://github.com/JKatheder/bisociation-net/issues/19 needs to be done first
